.github/workflows/instrumentation-tests.yml

```diff
@@ -0,0 +1,50 @@
+name: Instrumentation tests
+
+on: [push, pull_request]
+
+jobs:
+  instrumentation-test:
+    runs-on: macos-latest
+    timeout-minutes: 45
+    strategy:
+      # Add more samples here as they are added to the project.
+      matrix:
+        sample: [kotlin_compose, java_layout]
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up JDK 17
+      uses: actions/setup-java@v2
+      with:
+        distribution: 'adopt'
+        java-version: '17'
+
+    - name: Setup Android SDK
+      uses: android-actions/setup-android@v2
+
+    - name: Gradle cache
+      uses: gradle/gradle-build-action@v2
+
+    - name: Start emulator
+      uses: reactivecircus/android-emulator-runner@v2
+      with:
+        # Configuration options for the emulator for the best optimization.
```

Comment:
Can you link to the place you found these configuration settings? So we can try and keep them up-to-date?

Reply:
Configuration can be found action document, do you prefer me adding the link to [reactivecircus/android-emulator-runner@v2](https://github.com/ReactiveCircus/android-emulator-runner#configurations) github?

Reply:
I am trying to find where you found this configuration list *for the best optimization*. From [the docs](https://github.com/ReactiveCircus/android-emulator-runner#configurations), I see all the configuration options with default values but don't see where you came up with the configuration options you included. That's what I am looking for.

Reply:
Just read the description, took out the ones that aren't needed, and ran the workflow multiple times to test others.